db/db_impl/db_impl.h

```diff
@@ -355,6 +355,14 @@ class DBImpl : public DB {
 
   virtual bool SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) override;
 
+  // This functions can only be called when db_mutex is NOT held
+  Status IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                  std::string ts_low) override;
+
+  // This functions can only be called when db_mutex is NOT held
```

Comment:
I find the comments here a little confusing.
It says "This function can only be called when db_mutex is NOT held", but does this function not need db_mutex or will it acquire db mutex? I think it's better to explicitly specify as follows:
```c++
// IncreaseFullHistoryTsLow(ColumnFamilyHandle*, std::string) will acquire and release db_mutex
```
Similar applies to `GetFullHistoryTsLow()`.

Reply:
fixed